package.json

```diff
@@ -74,8 +74,11 @@
 		"@octokit/rest": "16.26.0",
 		"@storybook/addon-a11y": "5.2.3",
 		"@storybook/addon-docs": "5.2.3",
+		"@storybook/addon-storysource": "5.2.3",
 		"@storybook/addon-viewport": "5.2.3",
+		"@storybook/addons": "5.2.3",
 		"@storybook/react": "5.2.3",
+		"@storybook/source-loader": "5.2.3",
```

Comment:
Are you sure we need these two? The docs don't mention it.

Reply:
Removed and cleaned up packages which fixed other issue. 
I was testing numerous different ways and I think the package-lock.json got some weird stuff in it